lib/ansible/modules/cloud/smartos/smartos_image_info.py

```diff
@@ -33,21 +34,28 @@
 
 EXAMPLES = '''
 # Return facts about all installed images.
-- smartos_image_facts:
+- smartos_image_info:
 
 # Return all private active Linux images.
-- smartos_image_facts: filters="os=linux state=active public=false"
+- smartos_image_info: filters="os=linux state=active public=false"
 
 # Show, how many clones does every image have.
-- smartos_image_facts:
+- smartos_image_info:
+  register: result
 
-- debug: msg="{{ smartos_images[item]['name'] }}-{{smartos_images[item]['version'] }}
+- debug: msg="{{ result.smartos_images[item]['name'] }}-{{ result.smartos_images[item]['version'] }}
+            has {{ result.smartos_images[item]['clones'] }} VM(s)"
+  with_items: "{{ result.smartos_images.keys() }}"
```

Comment:
Do you mind also updating this to work with python 2 and 3?
```suggestion
  with_items: "{{ result.smartos_images.keys() | list }}"
```

Reply:
Good idea. Actually, shouldn't it also work without the `.keys()`? It is definitely more explicit with `.keys()`, though.

Reply:
True, good point.